README.md

```diff
@@ -1 +1,48 @@
 # Email Client
+
+## Tech Stack
+
+This project provides a few cool tools as a foundation, so it'd be worthwhile to check them out before getting started:
+
+- [Babel](http://babeljs.io/) for nextgen JavaScript features
+- [ESLint](https://eslint.org/) for help with JavaScript code errors and style
+- [Flow](https://flow.org/) for type-checking your JavaScript code
+- [Sass](http://sass-lang.com/) + [Autoprefixer](https://github.com/postcss/autoprefixer) for enhancing CSS with some nice features
+- [Stylelint](https://github.com/stylelint/stylelint) for help with CSS code errors and style
+- [Parcel](https://parceljs.org/) for building and running the app
+
+## Setup
+
+Getting the app installed isn't too scary! Just a few simple steps and you'll be all set:
+
+- Create an empty repo in your own GitHub account to house your copy of the app
+- Open a terminal/console and navigate to whatever folder you want the project to be in
+- Clone the sample repo with `git clone https://github.com/letscodework/email-client-practice-app.git`
+- Navigate into the downloaded folder with `cd email-client-practice-app`
+- Associate the Git info with yourself:
+  - `rm -rf .git`
+  - `git init`
+  - `git remote add origin YOUR_PROJECT_REPOSITORY_URL`
```

Comment:
Why not suggest they fork this repo instead of having to reinit `.git` and change remotes?